lib/workers/repository/index.ts

```diff
@@ -18,6 +23,7 @@ import { setBranchCache } from './cache';
 import { ensureDependencyDashboard } from './dependency-dashboard';
 import handleError from './error';
 import { finaliseRepo } from './finalise';
```

Comment:
`finalise` is the British spelling. We're following the American spelling rules on this repository, so I think it should be `finalize` with a `z`.

Do you want an issue from me to track the rename of the functions, constants, and directory names?

Reply:
sounds good to me